templates/core/terraform/scripts/build-deploy-ui.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
I think this file should move out of this folder and into the devops/scripts one as it has nothing to do with Terraform and called directly from the makefile.